region/brinstar/blue/Morph Ball Room.json

```diff
@@ -250,6 +246,239 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Power Beam Sidehopper Kill",
+      "requires": [
+        {"enemyDamage": {
+          "enemy": "Sidehopper",
+          "type": "contact",
+          "hits": 5
+        }}
+      ],
+      "clearsObstacles": ["C"]
+    },
+    {
+      "link": [1, 1],
+      "name": "Quick Sidehopper Kill",
+      "requires": [
+        {"enemyKill": {
+          "enemies": [
+            ["Sidehopper", "Sidehopper"],
+            ["Sidehopper"]
+          ],
+          "explicitWeapons": [
+            "Missile",
+            "Super",
+            "PowerBomb",
+            "ScrewAttack",
+            "Plasma"
+          ]
+        }}
+      ],
+      "clearsObstacles": ["C"],
+      "note": [
+        "For the Power Bomb and Screw Attack kills, this strat assumes you don't know you're entering the room beforehand.",
+        "If you know, it's possible to Power Bomb kill the first two Sidehoppers damage-free by morphing before entering.",
+        "For Screw, just entering with a spin jump would work."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Spark into Room",
+      "entranceCondition": {
+        "comeInWithSpark": {}
+      },
+      "requires": [
+        {"shinespark": {"frames": 40, "excessFrames": 9}}
+      ],
+      "clearsObstacles": ["C"]
+    },
+    {
+      "link": [1, 1],
+      "name": "Prepared Kill on Entry",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        {"or": [
+          {"and": [
+            "Plasma",
+            "Charge"
+          ]},
+          "ScrewAttack",
+          {"and": [
+            "h_canUsePowerBombs",
+            {"enemyKill": {
+              "enemies": [["Sidehopper"]],
+              "explicitWeapons": ["Missile", "Super", "PowerBomb"]
+            }}
+          ]}
+        ]}
+      ],
+      "clearsObstacles": ["C"],
+      "note": [
+        "Enter the room ready to kill the hoppers before they can hit Samus.",
+        "with a charge plasma shot ready, or in a screw attack spin jump, or a morph, ready to lay a power bomb."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Prepared Missile and Super Kill",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "canPrepareForNextRoom",
+        "canFarmWhileShooting",
```

Comment:
This is not a great tech here, but I was looking for something Expert-level that had some sort of enemy shooting/dodging.